pkgs/os-specific/linux/firmware/ath9k/urls-and-hashes-1.4.0.nix

```diff
@@ -0,0 +1,26 @@
+rec {
+  BASEDIR="$NIX_BUILD_TOP";
```

Comment:
Using NIX_BUILD_TOP should be avoided as much as possible. If it is embedded in the final result the build will fail.

Reply:
What is preferred instead?

Reply:
Usually using a relative path is preferred.